qemu/tests/cfg/live_backup.cfg

```diff
@@ -0,0 +1,36 @@
+- live_backup:
+    virt_test_type = qemu
+    type = live_backup
+    no vmdk qed
+    only Linux
```

Comment:
Get some error for md5 check part, and live backup whole feature is guest unrelated, so only Linux is enough to the feature.